### PR TITLE
Add kubernetes helm chart

### PR DESCRIPTION
**Pull request by yuanying**

### CODE REVIEW

File: kubernetes/chart/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Parameter | Description | Default
+--------- | ----------- | -------
+`replicaCount` | How many replicas to run. | 1
+`image.repository` | Name of the image to run, without the tag. | [hackmdio/hackmd](https://github.com/hackmdio/docker-hackmd)
+`image.tag` | The image tag to use. | 0.5.1
+`image.pullPolicy` | The kubernetes image pull policy. | IfNotPresent
+`service.name` | The kubernetes service name to use. | hackmd
+`service.type` | The kubernetes service type to use. | ClusterIP
+`service.externalPort` | Servuce exteral port. | 3000
+`service.internalPort` | Servuce internal port. | 3000
```
Comment (SISheogorath) [inline kubernetes/chart/README.md]: "Servuce" typo?

Comment (yuanying) [reply kubernetes/chart/README.md]: Fixed

**Comment by yuanying** [comment]
Hi, I'm using this for my private team development environment with Ingress and PVC support.
So, maybe this chart not for production but almost works well.

**Comment by SISheogorath** [comment]
I'll leave this open until Wednesday for annotations, comments, and reviews from people who are deeper into K8s and merge it when there are no further changes requested.

Thanks a lot for your work 👍

**Comment by SISheogorath** [comment]
Do we want to merge this or do you want to make a few more edits to it? I just saw you pushed a new commit.

**Comment by yuanying** [comment]
Thanks! It's ok to merge this patch.
Maybe there is no critical problem.

**Pull request merged**